artifact/v3/payload/payload.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <artifact/v3/payload/payload.hpp>
+
+#include <vector>
+
+#include <common/log.hpp>
+#include <common/io.hpp>
+#include <common/expected.hpp>
+
+namespace artifact {
+namespace v3 {
+namespace payload {
```

Comment:
I think it's covered in a different PR already, but should it be `mender::artifact::v3::payload`?